File: problems/python/ParkingLotSystem/src/strategy/payment/cash_payment_strategy.py
"""Cash payment strategy."""
from __future__ import annotations
from .payment_strategy import PaymentStrategy


class CashPaymentStrategy(PaymentStrategy):
    def pay(self, fee: float, ticket: "ParkingTicket") -> bool:
        print(f"Paid ₹{fee} for ticket {ticket.get_ticket_id()} via Cash.")
        return True

File: problems/java/ParkingLotSystem/src/strategy/payment/CashPaymentStrategy.java
package strategy.payment;

import model.ParkingTicket;

public class CashPaymentStrategy implements PaymentStrategy{
    @Override
    public boolean pay(double fee, ParkingTicket ticket) {
        System.out.println("Paid ₹" + fee + " for ticket " + ticket.getTicketId() + " via Cash.");
        return true;
    }
}
